lib/alephant/broker/cache/client.rb

```diff
@@ -11,7 +11,7 @@ class Client
 
         def initialize
           if config_endpoint.nil?
-            logger.debug 'Broker::Cache::Client#initialize: No config endpoint, NullClient used'
+            logger.error 'Broker::Cache::Client#initialize: No config endpoint, NullClient used'
```

Comment:
Can we standardise all of these calls, such as using brackets for all. Very minor and I know it was already like this, but good to get them consistent.